### DISCUSSION TITLE
参考文献refs和thu有什么区别？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#100] [Q&A]
你好，这两个有什么区别呢？为什么有两个呀，我该把文献放哪里？谢谢

**Comment by tanukihee** [comment]
> 这两个有什么区别呢

区别在于内容，你可以打开自己对照

> 为什么有两个呀

这只是示例 bib，旨在表明 whu-thesis 支持多 bib 功能

> 我该把文献放哪里

放在哪里都可以，只要你能引用得到就行

**Reply by earthmoon** [reply]
你好，那中英文混排，需要特殊标记吗？不然latex怎么知道那个是中文，哪个是英文呢？

**Reply by tanukihee** [reply]
不需要，现代 LaTeX 会根据字符的编码所在的范围判断到底是中文还是英文还是什么其他的东西，这些已经在 ctex 宏集中预定义好了

**Reply by earthmoon** [reply]
> 不需要，现代 LaTeX 会根据字符的编码所在的范围判断到底是中文还是英文还是什么其他的东西，这些已经在 ctex 宏集中预定义好了

这么强大，不可思议。
